packages/components/src/ui/styles/README.md

```diff
@@ -0,0 +1,61 @@
+# styles
+
+## The `ui` object
+
+The primary API that `styles` exposes is the `ui` object. It contains the following properties:
+
+- `get`: Retrieves a pre-defined set of CSS Custom Properties. See [Theme Configuration](#theme-configuration) below.
+- `tokens`: A pre-defined set of theme variables, accessible via the frozen `tokens` object. See [Theme Tokens](#theme-tokens) below.
+- Various mixins. See [Mixins](#mixins) below.
+
+## Theme Configuration
```

Comment:
Now that we're only keeping the variables that are already defined in our Sass files as theme configuration, does this mean that we can now remove all the code that generates the CSS variables from create-styles like discussed in the previous PR?

Reply:
No because the Sass variables are just Sass variables, they're not CSS custom properties so they're not accessible from CSS-in-JS. If anything I'd say we should remove the Sass variables and switch to just CSS custom properties?

Reply:
Now  I'm not talking about SASS variables, I'm talking about the CSS variables generated but the SASS mixin we already use to theme the components and UI.

Reply:
Gotcha. Currently that's only like one variable right? 

We already declare a dependency on `base-styles` anyway so I don't see that as a problem 🤔 But if we changed the style system to use the unstable prefix I think we should keep it consistent and keep all the style system variables inside the style system and not rely on anything external to generate the variables, right?'

Reply:
I think the current system in  create-styles that generates the variables on load from the theme is a bit too advanced for our needs. I don't really care whether we simplify that system and make it the canonical way of including the "default css variables" or whether we just enhance our existing mixin to include the new variables, but we should just have just one consistent system and use it across our packages. (components package not being the only package that can be themable and React dependency shouldn't be mandatory)

Reply:
To clarify, I don't think this is necessarily something that we need to act upon on this PR necessarily but something that need to be solved.